gpkit/constraints/relax.py

```diff
@@ -64,14 +76,31 @@ def __init__(self, constraints):
         if not isinstance(constraints, ConstraintSet):
             constraints = ConstraintSet(constraints)
         substitutions = dict(constraints.substitutions)
-        posynomials = constraints.as_posyslt1()
-        N = len(posynomials)
+        relconstrs = []
+        N = len(constraints)
```

Comment:
because N is only used once it probably isn't worth declaring as a variable...just stick it in the VectorVar call